Bot_project/main.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include <string>
+
+using namespace std; //I know using namespace std is bad practice, but i'm just testing stuff out.
+
+//a function that doesn't ask for a name
+//so I can call and return for a loop
+//(so the program stays awake).
+//Definitely not the best way but I'll change it later.
+
+int prompt(const string& input_name )
+{
+    //declaring the variables locally.
+    string input, answer;
+    string greetings[] = {"hello", "hi", "greetings", "how are you", "wussup"};
+
+    cout << "You: ";
+    getline(cin, input); // using *getline so it takes in strings with spaces.
+
+
+    //a for loop to turn Input into lowercase
+    for(char &input: input)
+        input = input | ' ';        // similar to: c = tolower(c);
+
+
+    //if statement to answer only to specific strings
+    if (input == greetings[0] || input == greetings[1])
+
+    {
+        answer =  "Hello there, " + input_name;
+        cout << "Bot: " << answer << endl;
+    }
+    else if (input == greetings[2])
+    {
+        answer = "Are you trying to be fancy? It's not working.";
+        cout << "Bot: " << answer << endl;
+    }
+    else if (input == greetings[3])
+    {
+        answer = "I'm fine, how can I help you?";
+        cout << "Bot: " << answer << endl;
+
+    }
+    else if (input == greetings[4])
+    {
+        answer = "Wussup homie.";
+        cout << answer << endl;
+    }
+    else
+    {
+        cout << "still in development!\n";
+    }
+
+
```

Comment:
convention is leave only one line between functions (yep, ppl are very picky :))